modules/update-preparing-evaluate-apirequestcount-workloads.adoc

```diff
@@ -41,7 +41,5 @@ $ oc get apirequestcounts csistoragecapacities.v1beta1.storage.k8s.io \
 [source,terminal]
 ----
 VERBS       USERNAME                        USERAGENT
-list watch  system:kube-controller-manager  cluster-policy-controller/v0.0.0
-list watch  system:kube-controller-manager  kube-controller-manager/v1.26.5+0abcdef
-list watch  system:kube-scheduler           kube-scheduler/v1.26.5+0abcdef
+// TODO: Need output, no hits in test cluster
```

Comment:
@wangke19 @sanchezl Do you know if any good example we can show here? My test cluster didn't have any hits, so I wasn't getting any output from this command.

Reply:
@sanchezl Any chance you could help with some example output here?

Reply:
```suggestion
VERBS     USERNAME                            USERAGENT
create    andrea                              Mozilla/5.0 (X11; Linux x86_64)
create    system:admin                        oc/4.13.0 (linux/amd64)
list get  system:serviceaccount:myns:default  oc/4.17.0 (linux/amd64)
watch     system:serviceaccount:myns:webhook  webhook/v1.0.0 (linux/amd64)
```